client/src/components/Projects/ColumnHeaderPopups/StatusPopup.js

```diff
@@ -4,6 +4,7 @@ import Button from "../../Button/Button";
 import RadioButton from "../../UI/RadioButton";
 import "react-datepicker/dist/react-datepicker.css";
 import { MdClose } from "react-icons/md";
+import UniversalSelect from "../../UI/UniversalSelect";
```

Comment:
I just figured out a day or two ago that the UniversalSelect control is not working.  If it is used as a controlled input, it does not not track the existing controlled value when it is first rendered.  We need to either fix it, or go back to using a standard react <select> control.

Reply:
I think this was what resulted in me going out of scope for this task. I discovered that when I attempted to change the sorting option, it would reset the filter option. I believe that the issue was that the StatusPopup component was being rendered each time, and so the existing controlled value was lost. I decided the best approach to fixing this was to save that value in state of the Project page.